paddle/phi/ops/compat/conv2d_sig.cc

```diff
@@ -53,9 +53,46 @@ KernelSignature Conv2dDoubleGradOpArgumentMapping(
                          {"DInput", "DFilter", "DDOutput"});
 }
 
+KernelSignature FusedConv2dOpArgumentMapping(
+    const ArgumentMappingContext& ctx) {
+  return KernelSignature("fused_conv2d",
+                         {"Input", "Filter", "Bias", "ResidualData"},
+                         {"strides",
+                          "paddings",
+                          "padding_algorithm",
+                          "dilations",
+                          "groups",
+                          "data_format",
+                          "mkldnn_data_type",
+                          "fuse_activation",
+                          "fuse_residual_connection",
+                          "force_fp32_output"},
+                         {"Output"});
+}
+
+KernelSignature FusedConv3dOpArgumentMapping(
+    const ArgumentMappingContext& ctx) {
+  return KernelSignature("fused_conv3d",
+                         {"Input", "Filter", "Bias", "ResidualData"},
+                         {"strides",
+                          "paddings",
+                          "padding_algorithm",
+                          "dilations",
+                          "groups",
+                          "data_format",
+                          "mkldnn_data_type",
+                          "fuse_activation",
+                          "fuse_residual_connection",
+                          "force_fp32_output"},
+                         {"Output"});
+}
+
```

Comment:
Shouldn't this be moved to _paddle/phi/ops/compat/conv3d_sig.cc_?

Reply:
Done. Add fused_conv_sig.cc for `fused_conv2d` and `fused_conv3d`.